rs/nns/governance/src/governance.rs

```diff
@@ -462,7 +462,7 @@ impl ManageNeuronResponse {
         !self.is_err()
     }
 
-    pub fn expect(self, msg: &str) -> Self {
+    pub fn panic_if_error(self, msg: &str) -> Self {
```

Comment:
Note, this is changed because it does not function the same way as ".expect()" which unwraps an option or Result, but rather just panics if the response type is an error.